po/common/common.pot

```diff
@@ -5428,3 +5428,207 @@ msgstr "Type of the product photo"
+msgstr ""
+
+msgctxt "ecoscore_missing_agribalyse_match_warning"
+msgid "<strong>The Eco-Score can only be calculated if the product has a sufficiently precise category.</strong><br><br>You can modify the product page to add a more precise category.<br><br>If you are the manufacturer of this product, you can send us the information with our <a href=\"https://world.pro.openfoodfacts.org\">free platform for producers</a>."
+msgstr ""
+
+# TODO DEDUPLICATE SOME STRINGS, CHECK THAT msgctxt are unique
+
+msgctxt "ecoscore_product_category_reference_score"
+msgid "Baseline score of the product category"
+msgstr ""
+
+msgctxt "ecoscore_panel_lca"
+msgid "Lifecyle Analysis (LCA)</h4>
+msgstr ""
+
+msgctxt "agribalyse_category"
+msgid "Agribalyse category"
+msgstr ""
+
+msgctxt "category_proxy_match"
+msgid "Approximate match with the product category"
+msgstr ""
+
+msgctxt "category_exact_match"
+msgid "Exact match with the product category"
+msgstr ""
+
+msgctxt "pef_environmental_score"
+msgid "PEF environmental score:"
+msgstr ""
+
+msgctxt "incl_climate_change_impact"
+msgid "- including impact on climate change:"
+msgstr ""
+
+msgctxt "impact_detail_by_stages"
+msgid "Details of the impacts by stages of the life cycle"
+msgstr ""
+
+msgctxt "stage"
+msgid "Stage"
+msgstr ""
+
+msgctxt "impact"
+msgid "Impact"
+msgstr ""
+
+msgctxt "agriculture"
+msgid "Agriculture"
+msgstr ""
+
+msgctxt "transformation"
+msgid "Transformation"
+msgstr ""
+
+msgctxt "packaging"
+msgid "Packaging"
+msgstr ""
+
+msgctxt "transportation"
+msgid "Transportation"
+msgstr ""
+
+msgctxt "distribution"
+msgid "Distribution"
+msgstr ""
+
+msgctxt "consumption"
+msgid "Consumption"
+msgstr ""
+
+msgctxt "lca_score_on_100"
+msgid "LCA score on 100:"
+msgstr ""
+
+msgctxt "no_agribalyse_category_match"
+msgid "No match between product categories and Agribalyse categories."
+msgstr ""
+
+msgctxt "edit_category_to_more_granular"
+msgid "You can modify the product page to add a more granular category."
+msgstr ""
+
+msgctxt "additional_bonuses_and_maluses"
+msgid "Additional bonuses and maluses"
+msgstr ""
+
+msgctxt "production_mode"
+msgid "Production mode"
+msgstr ""
+
+msgctxt "label"
+msgid "Label"
+msgstr ""
+
+msgctxt "no_labels_taken_into_account"
+msgid "No labels taken into account for the production system."
+msgstr ""
+
+msgctxt "please_add_the_labels"
+msgid "If this product has a label characterizing the production system (organic, fair trade, Label Rouge, Bleu Blanc Coeur etc.), you can modify the product sheet to add it."
+msgstr ""
+
+msgctxt "origin_of_ingredients"
+msgid "Origin of ingredients"
+msgstr ""
+
+msgctxt "ingredients_not_indicated"
+msgid "The origins of the ingredients of this product are not indicated."
+msgstr ""
+
+msgctxt "please_add_the_ingredients"
+msgid "If they are indicated on the packaging, you can modify the product sheet and add them."
+msgstr ""
+
+msgctxt "environmental_policy"
+msgid "Environmental policy :"
```

Comment:
We should remove the : and put it in the template, so that it can have a space separator in some languages (but not English).

Reply:
and that's probably why it should be left in the string, for the translators to decide. Or better yet, something like:
msgid "Environmental policy : %s"

Reply:
we have defined a separator character for each language, so that it's uniform.

Reply:
Currently in the strings you added there's a mix of  "Something:" and "Something else :".

In the template, you can use $sep , it will be empty for almost all languages, except French where it will be a non breaking space.